authentication_service/example_config

```diff
@@ -0,0 +1,6 @@
+port: ":your_port_number"
```

Comment:
Can you add instructions on what must be configured in order for this application to run? E.g. in a docs page or readme. Also, can you add an example configuration file with values filled in so we know how the values must be formatted?